src/bundle/Resources/public/scss/ui/modules/universal-discovery/_top.menu.search.input.scss

```diff
@@ -0,0 +1,34 @@
+.c-top-menu-search-input {
+    &--search-opened {
+        .c-top-menu-search-input {
+            &__search-input {
+                opacity: 1;
+                transform: translateY(-50%) scaleX(1);
+            }
+        }
+    }
+
+    &__search-btn {
+        transition: none;
+    }
+
+    &__search-input {
+        position: absolute;
+        width: calc(100% - #{calculateRem(224px)});
+        top: 50%;
+        left: calculateRem(65px);
+        border: none;
+        padding: 0;
+        font-family: $headings-font-family;
+        font-weight: 600;
+        font-size: $ibexa-text-font-size-extra-large;
+        opacity: 0;
+        transform: translateY(-50%) scaleX(0);
+        transform-origin: left;
+        transition: all $ibexa-admin-transition-duration $ibexa-admin-transition, transform 0s;
```

Comment:
Weren't there some problems with 0s? I remember Darek told sth like this this week

@dew326

Reply:
`0` is not working, you need to add `s` at the end.